docs/PORTING.md

```diff
@@ -1,36 +1,43 @@
-# Porting How-To
+# Porting how-to
 
-This document describes the requirements and necessary steps required to port
-`mcuboot` to a new target `OS`.
+This document describes the requirements and necessary steps needed to
```

Comment:
"the necessary"

Reply:
While rereading the page I realized that, in this case, being this the first sentence of the page, the reader knows neither of the two yet. Dropping "The" from both